### DISCUSSION TITLE
How can i deal with Postgresql Views

### DISCUSSION THREAD

**Discussion by Temkit** [keystonejs/keystone#5889] [Questions]
Hi, 

i am creating a KeystoneJS inventory Management System, and need the view Feature of Postgres to display to real time inventory from two tables : INs and OUTs ... 

How can we do that in keystoneJS Next ?

**Comment by bladey** [comment]
Hey @Temkit, GitHub Discussions is currently used for feature requests only, if this is still relevant please join us on https://community.keystonejs.com!
